.github/workflows/ios.yml

```diff
@@ -46,3 +46,9 @@ jobs:
     - name: Tests
       working-directory: platforms/ios/example
       run: exec ./ios-tests.sh
+
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v3
+      with:
+        fail_ci_if_error: true
```

Comment:
I suppose it should have `directory: platforms/ios/example/DerivedData` (or something slightly deeper in that hierarchy) since we are saving to a custom derived data folder.

Reply:
Codecov doesn’t support the apple format for coverage. So I have some changes to integrate slather, the same library used on the main app.